### PR TITLE
Implemented careers and careers to skills

### PR DESCRIPTION
**Pull request by lawsonbh**
I think the users will really enjoy seeing this and this gives us more to work with as I move into the creating the :mage: wizard experience.

Implemented a model for Careers and linked it to skills with an assigned modifier through an intermediary model.

Included the migration file and an updated seed data file.

### CODE REVIEW

File: bodybank/character_generator/models.py [hunk 1]
```diff
@@ -107,4 +107,20 @@ class Background2Skills(models.Model):
...
+    name = models.CharField(max_length=100)
+    desc = models.TextField("Description", default="")
+
+    def __str__(self):
+        return self.name
+
+class Career2Skills(models.Model):
+    career = models.ForeignKey(Career, related_name = 'career_to_skill', on_delete=models.CASCADE)
+    skills = models.ForeignKey(Skill, related_name='skill_to_career', on_delete=models.CASCADE)
+    modifier = models.PositiveIntegerField(default=0)
```
Comment (bbelderbos) [inline bodybank/character_generator/models.py:123]: is there a better name for this or is it called like that in the game?

File: bodybank/character_generator/models.py [hunk 2]
```diff
@@ -107,4 +107,20 @@ class Background2Skills(models.Model):
...
+
+    def __str__(self):
+        return (' '.join([str(self.career), str(self.skills), str(self.modifier)]))
```
Comment (bbelderbos) [inline bodybank/character_generator/models.py:126]: drop `()`

File: bodybank/character_generator/models.py [hunk 3]
```diff
@@ -107,4 +107,20 @@ class Background2Skills(models.Model):
     modifier = models.PositiveIntegerField(default=0)
 
     def __str__(self):
-        return (' '.join([str(self.background), str(self.skills), str(self.modifier)]))
\ No newline at end of file
+        return (' '.join([str(self.background), str(self.skills), str(self.modifier)]))
+
+
+class Career(models.Model):
```
Comment (bbelderbos) [inline bodybank/character_generator/models.py:113]: Why did you add more models? I am ok with it as long as we don't delay too much on the wizard which is a core piece of functionality

**Review by bbelderbos — APPROVED** [review]

File: bodybank/character_generator/models.py (hunk 1 above)
Comment (lawsonbh) [reply bodybank/character_generator/models.py:123]: Its called career in the game :raccoon:

File: bodybank/character_generator/models.py (hunk 3 above)
Comment (lawsonbh) [reply bodybank/character_generator/models.py:113]: I thought the users would like it and I was able to use the same pattern as I did for background so it was a quick add :raccoon:
